modules/030-cloud-provider-azure/templates/node-termination-handler/daemonset.yaml

```diff
@@ -0,0 +1,80 @@
+{{- if (.Values.global.enabledModules | has "vertical-pod-autoscaler-crd") }}
+---
+apiVersion: autoscaling.k8s.io/v1
+kind: VerticalPodAutoscaler
+metadata:
+  name: node-termination-handler
+  namespace: d8-{{ .Chart.Name }}
+  {{- include "helm_lib_module_labels" (list . (dict "app" "node-termination-handler" "workload-resource-policy.deckhouse.io" "every-node")) | nindent 2 }}
+spec:
+  targetRef:
+    apiVersion: "apps/v1"
+    kind: DaemonSet
+    name: node-termination-handler
+  updatePolicy:
+    updateMode: "Auto"
+{{- end }}
+---
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: node-termination-handler
+  namespace: d8-{{ .Chart.Name }}
+  {{- include "helm_lib_module_labels" (list . (dict "app" "node-termination-handler")) | nindent 2 }}
+spec:
+  {{- include "helm_lib_pdb_daemonset" . | nindent 2}}
+  selector:
+    matchLabels:
+      app: node-termination-handler
+---
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: node-termination-handler
+  namespace: d8-{{ .Chart.Name }}
+  {{- include "helm_lib_module_labels" (list . (dict "app" "node-termination-handler")) | nindent 2 }}
+spec:
+  selector:
+    matchLabels:
+      app: node-termination-handler
+  template:
+    metadata:
+      labels:
+        app: node-termination-handler
+    spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - operator: In
+                key: node.deckhouse.io/type
+                values: ["CloudEphemeral"]
+      imagePullSecrets:
+      - name: deckhouse-registry
+      {{- include "helm_lib_priority_class" (tuple . "system-node-critical") | nindent 6 }}
+      {{- include "helm_lib_tolerations" (tuple . "any-node") | nindent 6 }}
+      {{- include "helm_lib_module_pod_security_context_run_as_user_nobody" . | nindent 6 }}
+      serviceAccountName: node-termination-handler
+      hostNetwork: true
+      dnsPolicy: ClusterFirstWithHostNet
+      containers:
+      - name: node-termination-handler
+        {{- include "helm_lib_module_container_security_context_read_only_root_filesystem" . | nindent 8 }}
+        image: "{{ .Values.global.modulesImages.registry }}:{{ .Values.global.modulesImages.tags.cloudProviderAzure.nodeTerminationHandler }}"
+        env:
+        - name: NODE_NMY_NODE_NAMEAME
```

Comment:
Strange name. Are you sure it's not a mistake?

Reply:
Definitely a mistake. Thanks!

https://github.com/maksim-paskal/aks-node-termination-handler/blob/4de9ead3c97a962c80bb177caa2dbcdb5ea78a63/pkg/config/config.go#L67